src/table-list-field/table-list-field.component.html

```diff
@@ -6,37 +6,17 @@
           <th *ngFor="let key of keys | addAlwaysShowFields:schema.items | filterAndSortBySchema:schema.items; trackBy:trackByElement">
             {{key | underscoreToSpace}}
           </th>
-
           <th class="button-holder">
             <add-field-dropdown *ngIf="values.size > 0" [fields]="keys" [pathString]="getElementPathString(0)" (onFieldAdd)="onFieldAdd($event)"
               [schema]="schema.items.properties">+</add-field-dropdown>
           </th>
-
         </tr>
       </thead>
-      <tr *ngFor="let row of values; let i = index; trackBy:trackByIndex" [id]="getElementPathString(i)">
-        <!-- Element value -->
-        <td *ngFor="let key of keys | addAlwaysShowFields:schema.items | filterAndSortBySchema:schema.items; trackBy:trackByElement"
-          [style.width]="schema.items.properties[key].columnWidth + '%'">
-          <any-type-field [value]="row.get(key) | selfOrEmpty:schema.items.properties[key]" [schema]="schema.items.properties[key]"
-            [path]="getValuePath(i, key)"></any-type-field>
-            <add-new-element-button *ngIf="schema.items.properties[key].type === 'array'" [path]="getValuePath(i, key)" [schema]="schema.items.properties[key]"></add-new-element-button>
-            </td>
-
-            <!-- UP/DOWN and DELETE buttons for each row -->
-            <td *ngIf="values.size > 0" class="button-holder">
-              <button type="button" class="editor-btn-delete" (click)="deleteElement(i)">
-            &times;
-          </button>
-              <button *ngIf="i > 0" type="button" class="editor-btn-move-up" (click)="moveElement(i, -1)">
-            <i class="fa fa-chevron-up" aria-hidden="true"></i>
-          </button>
-              <button *ngIf="i < (values.size - 1)" type="button" class="editor-btn-move-down" (click)="moveElement(i, 1)">
-            <i class="fa fa-chevron-down" aria-hidden="true"></i>
-          </button>
-            </td>
-      </tr>
-      <tr>
+      <tr *ngFor="let value of values; let i = index; trackBy:trackByIndex" table-item-field [id]="getElementPathString(i)" [value]="value"
```

Comment:
You can set a component selector inside an html tag? So, in this case what `ngFor` will produce as template?